docs/usage/shoot-operations/shoot_updates.md

```diff
@@ -53,21 +53,45 @@ We consider shoot upgrades to change either the:
 Generally, an upgrade is also performed through a reconciliation of the `Shoot` resource, i.e., the same concepts as for [shoot updates](#updates) apply.
 If an end-user triggers an upgrade (e.g., by changing the Kubernetes version) after a new Gardener version was deployed but before the shoot was reconciled again, then this upgrade might incorporate the changes delivered with this new Gardener version.
 
-### In-Place vs. Rolling Updates
+`UpdateStrategy` field under shoot worker enables the end user to choose between different behaviour of machine-controller-manager (abbrev.: MCM; the component instrumenting this rolling update) during the upgrade process. Currently gardener support three update strategies:
+* `AutoRollingUpdate`
+* `AutoInPlaceUpdate`
+* `ManualInPlaceUpdate`
 
-If the Kubernetes patch version is changed, then the upgrade happens in-place.
-This means that the shoot worker nodes remain untouched and only the `kubelet` process restarts with the new Kubernetes version binary.
-The same applies for configuration changes of the kubelet.
+### In-Place vs. Rolling Updates
 
-If the Kubernetes minor version is changed, then the upgrade is done in a "rolling update" fashion, similar to how pods in Kubernetes are updated (when backed by a `Deployment`).
-The worker nodes will be terminated one after another and replaced by new machines.
-The existing workload is gracefully drained and evicted from the old worker nodes to new worker nodes, respecting the configured `PodDisruptionBudget`s (see [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)).
+In case of in-place update worker nodes are updated without the need for replacing the VMs with the new ones. This means that the shoot worker nodes remain untouched. There is some downtime expected as the node needs to be drained and then workload will be scheduled again once the node is updated. If the Kubernetes patch version is changed, then the upgrade happens in-place. This means that the shoot worker nodes remain untouched and only the kubelet process restarts with the new Kubernetes version binary. The same applies for configuration changes of the kubelet.
+
+In case of rolling update, upgrade is done in a "rolling update" fashion, similar to how pods in Kubernetes are updated (when backed by a `Deployment`).The worker nodes will be terminated one after another and replaced by new machines.The existing workload is gracefully drained and evicted from the old worker nodes to new worker nodes, respecting the configured `PodDisruptionBudget`s (see [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)).
```

Comment:
```suggestion
In case of rolling update, upgrade is done in a "rolling update" fashion, similar to how pods in Kubernetes are updated (when backed by a `Deployment`). The worker nodes will be terminated one after another and replaced by new machines. The existing workload is gracefully drained and evicted from the old worker nodes to new worker nodes, respecting the configured `PodDisruptionBudget`s (see [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)).
```